CHANGELOG.md

```diff
@@ -1,5 +1,36 @@
 # [Versions](https://mui.com/versions/)
 
+## 6.3.1
+
+<!-- generated comparing v6.3.0..master -->
+
+_Jan 2, 2025_
+
+A big thanks to the 7 contributors who made this release possible. Here are some highlights ✨:
+
+### `@mui/material@6.3.1`
+
+- [Autocomplete] Revert: Fix options list rendering in freeSolo mode (#44858) @ZeeshanTamboli
```

Comment:
Just to double check: are we ok with this being released in a patch and not a minor?

Reply:
6.3.1 is the correct version to use. A revert of a previous change is considered a bug fix (patch).

Reply:
What's the impact to users?

Reply:
Explained here: https://github.com/mui/material-ui/pull/44858#issuecomment-2568728703